### PR TITLE
[CS] Add `Gameplay/Choke`

### PR DESCRIPTION
**Pull request by RadekMojzis**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Choke/cs.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+tags:
+  - reverse choke
+---
+
+# Choke
+
+Termín používaný, když se hráči skoro podaří udržet plné kombo když hraje [beatmapu](/wiki/Beatmap), ale kombo přeruší blízko konce mapy.
+**Obrácený choke** potom referuje na situaci, kdy hráč přeruší kombo blízko začátku mapy a následně si jej udrží až do konce.
+V [osu!](/wiki/Game_mode/osu!) a [osu!catch](/wiki/Game_mode/osu!catch) může být rozdíl mezi chokem (nebo reverse chokem) a plným kombem poměrně drastický. Je to z toho důvodu, že objekty blízko konce mají vysokou váhu kvůli akumulovanému kombu.
```
Comment (TPGPL) [inline wiki/Gameplay/Choke/cs.md:10]:
```suggestion
Termín používaný, když se hráči skoro podaří udržet plné kombo když hraje [beatmapu](/wiki/Beatmap), ale kombo přeruší blízko konce mapy. **Obrácený choke** potom referuje na situaci, kdy hráč přeruší kombo blízko začátku mapy a následně si jej udrží až do konce. V [osu!](/wiki/Game_mode/osu!) a [osu!catch](/wiki/Game_mode/osu!catch) může být rozdíl mezi chokem (nebo reverse chokem) a plným kombem poměrně drastický. Je to z toho důvodu, že objekty blízko konce mají vysokou váhu kvůli akumulovanému kombu.
```

File: wiki/Gameplay/Choke/cs.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Tento termín také používá velké množství jiných komunit, kde má zpravidla stejný, nebo podobný význam. (například [*Super Smash Bros.*](https://en.wikipedia.org/wiki/Super_Smash_Bros.) nebo [*speedrunning*](https://en.wikipedia.org/wiki/Speedrun))
+
+## Co způsobuje choke
+
+Klíčovými vlastnostmi choku jsou náhoda a nekonzistentní výkon. Přerušení komba na specifickém obtížném vzoru (paterně) může být zaviněno nedostatečnými dovednostmi hráče, nebo způsobem jakým byla beatmapa konstruována. Choke je většinou způsobený nadměrnou psychickou zátěží, nervozitou z hraní a celkovou únavou.
```
Comment (TPGPL) [inline wiki/Gameplay/Choke/cs.md:16]: missing a link

for ref:

`The key characteristic of a choke is its randomness and inconsistent performance. This excludes regular or pattern-specific misses or [slider breaks](/wiki/Gameplay/Judgement/Slider_break), which are usually tied to a player's skill level, or how the beatmap is constructed. Chokes are typically caused by excessive mental stress and nervousness during gameplay, as well as overall fatigue.`

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Choke/cs.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+tags:
+  - reverse choke
+---
+
+# Choke
+
+Termín používaný, když se hráči skoro podaří udržet plné kombo když hraje [beatmapu](/wiki/Beatmap), ale [kombo přeruší](/wiki/Gameplay/Judgement/Combobreak) blízko konce mapy. **Obrácený choke** potom referuje na situaci, kdy hráč přeruší kombo blízko začátku mapy a následně si jej udrží až do konce. V [osu!](/wiki/Game_mode/osu!) a [osu!catch](/wiki/Game_mode/osu!catch) může být rozdíl mezi chokem (nebo reverse chokem) a plným kombem poměrně drastický. Je to z toho důvodu, že objekty blízko konce mají vysokou váhu kvůli akumulovanému kombu.
```
Comment (TicClick) [inline wiki/Gameplay/Choke/cs.md:8]: the third sentence is missing a mention of total score:

> ```md
> In [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch), the difference in [total score](/wiki/Gameplay/Score) between regular and reverse chokes may be drastic
> ```

**Review by TicClick — CHANGES_REQUESTED** [review]

**Comment by TPGPL** [comment]
converting to draft since there has been no activity for a long time

**Pull request closed without merging**